src/cc_catalog_airflow/dags/provider_api_scripts/rawpixel.py

```diff
@@ -0,0 +1,182 @@
+from common.storage.image import ImageStore
+from common.requester import DelayedRequester
+import requests
+import time
+import logging
+import json
+from urllib.parse import urlparse, parse_qs
+
+DELAY = 1.0  # time delay (in seconds)
+PROVIDER = "rawpixel"
+FILE = "rawpixel_{}.tsv".format(int(time.time()))
+
+logging.basicConfig(
+    format="%(asctime)s - %(name)s - %(levelname)s:  %(message)s",
+    level=logging.INFO,
+)
+
+logger = logging.getLogger(__name__)
+
+delayed_requester = DelayedRequester(DELAY)
+image_store = ImageStore(provider=PROVIDER, output_file=FILE)
+
+
+def request_content(_url, _query_params=None, _headers=None):
```

Comment:
Why the leading underscores on the function arguments?  They're kind of 'public' by definition.